build/verify-features.ts

```diff
@@ -53,12 +55,29 @@ function findError(filename: string): string | void {
 
 	const featureMeta = featuresInReadme.find(feature => feature.id === featureId);
 	if (!featureMeta) {
-		return `ERR: The feature ${featureId} should be described in the readme`;
+		return `ERR: ${featureId} should be described in the readme`;
 	}
 
 	if (featureMeta.description.length < 20) {
 		return `ERR: ${featureId} should be described better in the readme (at least 20 characters)`;
 	}
+
+	const lineRegex = regexJoin(/^/, `- [](# "${featureId}")`, /(?: 🔥)? (.+)$/gm);
+	const imageRegex = regexJoin(`<p><a title="${featureId}"></a> `, /(.+?)\n\t+<p><img src="(.+?)">/);
+	const lineMatch = readmeContent.match(lineRegex);
+	const imageMatch = readmeContent.match(imageRegex);
+	if (
+		(lineMatch && lineMatch.length > 1) // If the description occurs more than once in the large list of features
+		|| (imageMatch && lineMatch) // If the description appears in both the feature list and the highlighted features section
+	) {
+		const matches = readmeContent.split(/\r?\n/).map((lineContent: string, lineNumber: number) =>
+			lineRegex.test(lineContent)
+				|| regexJoin(`<p><a title="${featureId}"></a> `).test(lineContent)
+				? lineNumber + 1 : -1,
+		).filter(lineNumber => lineNumber > 0);
```

Comment:
I don’t think we need the like number, that's easy to find once you know there's a duplicate

Reply:
Commented it out for now, let me know if I should remove it completely or keep it.

IMO it's a good feature, saves the effort of a manual ctrl/cmd+f.